style.css

```diff
@@ -671,3 +735,336 @@
+  }
+
+  .slider h5 {
+    margin-left: 260px;
+  }
+
+  .slider .social-links ul {
+    margin-left: 260px;
+  }
+
+  .container {
+    width: 100%;
+    height: 2800px;
+  }
+
+  .work1 {
+    height: 496px;
+    width: 1156px;
+    display: flex;
+  }
+
+  .work1-text {
+    display: flex;
+    flex-direction: column;
+    margin-left: 22px;
+    margin-top: 48px;
+  }
+
+  .workimg {
+    margin-left: 24px;
+    margin-top: 24px;
+    margin-right: 24px;
+  }
+
+  .workimg img {
+    width: 544px;
+    height: 448px;
+  }
+
+  #work1-img-mob,
+  #work2-img-mob,
+  #work3-img-mob,
+  #work4-img-mob {
+    display: none;
+  }
+
+  .items h1 {
+    font-size: 40px;
+    line-height: 52px;
+    width: 496px;
+    height: 46px;
+  }
+
+  .items h2 {
+    font-size: 18px;
+    line-height: 24px;
+    margin-top: 20px;
+    width: 75px;
+    height: 24px;
+  }
+
+  .items h3 {
+    font-size: 18px;
+    line-height: 24px;
+    margin-top: 28px;
+    width: 200px;
+    height: 24px;
+  }
+
+  .items p {
+    font-size: 16px;
+    line-height: 24px;
+    width: 496px;
+    height: 50px;
+    margin-top: 32px;
+    padding-top: 0.1px;
+  }
+
+  .work-tags {
+    padding-top: 0.1px;
+  }
+
+  .button {
+    margin-top: 40px;
+  }
+
+  .work2 {
+    height: 496px;
+    width: 1156px;
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .work2-text {
+    display: none;
+  }
+
+  .work4-text {
+    display: none;
+  }
+
+  .work2-text-768 {
+    display: flex;
+    margin-left: 22px;
+    margin-top: 48px;
+    width: 544px;
+    height: 448px;
+    flex-direction: column;
+  }
+
+  .work3 {
+    height: 496px;
+    width: 1156px;
+    display: flex;
+  }
+
+  .work3-text {
+    display: none;
+  }
+
+  .work3-text-768 {
+    display: flex;
+    flex-direction: column;
+    margin-left: 22px;
+    margin-top: 48px;
+  }
+
+  .work4 {
+    height: 496px;
+    width: 1156px;
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  .work4-text-768 {
+    display: flex;
+    margin-left: 22px;
+    margin-top: 48px;
+    width: 544px;
+    height: 448px;
+    flex-direction: column;
+  }
+
+  .work4-text-768 h2 {
+    width: 20px;
+  }
+
+  .work4-text-768 h3 {
+    width: 205px;
+  }
+
+  #handcursor {
+    position: absolute;
+    width: 57px;
+    height: 57px;
+    margin-left: 400px;
+    margin-top: 320px;
+  }
+
+  .about-bg {
+    width: 100%;
+    border-top-right-radius: 80px;
+    display: flex;
+    height: 740px;
+  }
+
+  .about-myself {
+    width: 100%;
+    display: flex;
+    height: 738px;
+  }
+
+  .about-myself-upper {
+    display: flex;
+    flex-direction: column;
+    width: 500px;
+    height: 650px;
+    margin-top: 35px;
+    margin-left: 142px;
+  }
+
+  .about-myself-lower {
+    display: flex;
+    flex-direction: column;
+    width: 500px;
+    height: 650px;
+    margin-top: 110px;
+  }
+
+  .about-text {
+    width: 447px;
+    height: 102px;
+  }
+
+  .about-myself-upper h2 {
+    padding-top: 45px;
+  }
```

Comment:
Your about section looks a bit different from the design😉 I think we can make some comparison with the image below and add the necessary changes😉 don't forget, it is a requirement that we stick to the design😉

your app
<img width="946" alt="your___about" src="https://user-images.githubusercontent.com/64643995/213680207-185a8ef8-d331-45e0-b9f1-d87ce0fd453f.png">

design
<img width="572" alt="design__about" src="https://user-images.githubusercontent.com/64643995/213681162-09a45024-2bc0-4776-90d8-11af7b727981.png">